ci.rb

```diff
@@ -180,11 +180,18 @@ def chromedriver_uri
 end
 
 dep "chromium-browser.bin", :version do
-  version.default!("64")
+  version.default!("65")
 
   met? do
     in_path? "chromium-browser >= #{version}"
   end
+
+  meet {
+    log_shell "Guarantee we have add-apt-repository command", "apt-get --yes install software-properties-common", spinner: true, sudo: true
+    log_shell "Adding Chromium apt-repository", "add-apt-repository \"ppa:canonical-chromium-builds/stage\"", spinner: true, sudo: true
+    log_shell "Updating apt lists to load Chromium.", "apt-get update", sudo: true
+    log_shell "Download latest chromium", "apt-get --yes install chromium-browser", sudo: true
+  }
```

Comment:
Downloading  `software-properties-common` in this step didn't seem right, but not sure where to put it. Where should I put it? (cc @nullobject)

Reply:
Unless we're using it somewhere else I wouldn't bother pulling it up into another dep.